DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Plugin/RevitExternalApplication.cs

```diff
@@ -72,16 +83,22 @@ public Result OnStartup(UIControlledApplication application)
     return Result.Succeeded;
   }
 
+  static IEnumerable<DLlVersion> Walk(DLlVersion dllv)
+  {
+    var nodes = new Stack<DLlVersion>(new[] { dllv });
+    while (nodes.Any())
+    {
+      DLlVersion node = nodes.Pop();
+      yield return node;
+      foreach (var n in node.dependencyChains)
+        nodes.Push(n);
+    }
+  }
```

Comment:
What is this doing? 🤔

Reply:
Nothing thought I'd also removed this. I did look :/